File: src/chapters/chapter-05/csharp/Models/Product.cs
using System.ComponentModel;

namespace AdvancedAIShoppingAssistant.Models;

public class Product
{

    [Description("name")]
    public string? Name { get; set; } = "";

    [Description("Product code.")]
    public string? ProductCode { get; set; } = "";

    [Description("Product description.")]
    public string? Description { get; set; } = "";

    [Description("Product price.")]
    public decimal? Price { get; set; } = 0;

    [Description("Product category.")]
    public string? Category { get; set; } = "";

    [Description("Product brand.")]
    public string? Brand { get; set; } = "";
}

File: src/chapters/chapter-05/python/models/product.py
from dataclasses import dataclass
from semantic_kernel.functions import kernel_function
from typing import List, Optional
from dataclasses import dataclass, field

@dataclass
class Product:
    name: str = field(metadata={"description": "Short name of the product."})
    category: str = field(metadata={"description": "Product category."})
    description: str = field(metadata={"description": "Description of the product."})
    price: float = field(metadata={"description": "Price of the product."})
    product_code: str = field(metadata={"description": "Unique product code."})
    brand : str = field(metadata={"description": "Brand of the product."})